### DISCUSSION TITLE
Hello, help to obtain dimensions of a Formula

### DISCUSSION THREAD

**Discussion by hpvcodeDev** [ForNeVeR/wpf-math#479] [Q&A] [answered]
Hello, I have been trying to obtain the height and width dimensions of a formula, I got it this way.

`        public (double width, double height) GetTextDimensions(string textoIngresado)
        {
            var parser = WpfTeXFormulaParser.Instance;
            var formula = parser.Parse(textoIngresado);
            var environment = WpfTeXEnvironment.Create(TexStyle.Display, 12, "Arial");
            var bitmapSource = formula.RenderToBitmap(environment);

            double width = bitmapSource.Width;
            double height = bitmapSource.Height;

            return (width, height);
        }` 
the problem is that WpfTeXEnvironment.Create(TexStyle.Display, 12, "Arial"); in scale size change to any value, by default it is 20 from the same WpfMath code so it will always return the dimensions in Scala 20. Well, I don't know if there is another way to obtain the dimensions of the formula by changing the custom scale .

**Comment by ForNeVeR** [comment, marked as answer]
Try additionally passing the scale to `RenderToBitmap`, e.g. `formula.RenderToBitmap(environment, scale: 12.0)`.

It is known to be a bit confusing that the scale is used in two different places. Perhaps there's a room for improvement.

**Reply by hpvcodeDev** [reply]
Thank you very much, that is what I needed formula.RenderToBitmap(environment, scale: 12.0) when modifying the scale values ​​now it already returns the dimensions I need. Thank you.
